public/data/bisesh.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "Bisesh Adhikari",
+  "type": "personal",
+  "bio": "Student | Tech enthusiast | Aspiring software developer",
+  "avatar": "https://avatars.githubusercontent.com/u/99080674?v=4",
```

It is recommended to use the profile photo from GitHub using your username directly. That way, it'll be updated when you update your profile photo automatically. 😄
```suggestion
  "avatar": "https://github.com/Biseshadhikari.png",
```

---

Sorry, I just copied the URL of my profile photo. I will resolve it right away.
Thank you!!